cast/tests/e2e/helpers/devnet.rs

```diff
@@ -0,0 +1,43 @@
+use ctor::{ctor, dtor};
+use std::process::{Command, Stdio};
+use std::string::ToString;
+use std::thread;
+use std::time::Duration;
+use url::Url;
+
+pub const URL: &str = "http://127.0.0.1:5055/rpc";
```

Comment:
I have picked it.
Devnet starts by default with `5050`, that's why I have picked some other ports - to avoid clashes if someone has devnet already started for some other things.